modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -129,6 +129,63 @@ G_TYPED_KERNEL(GBackgroundSubtractor, <GMat(GMat, BackgroundSubtractorParams)>,
     }
 };
 
+/** @brief Structure for the Kalman filter's initialization parameters.*/
+
+struct GAPI_EXPORTS KalmanParams
```

Comment:
Please move this structure from video namespace (should be cv::gapi)

Reply:
Done.